src/main/java/net/sf/mzmine/modules/visualization/spectra/simplespectra/spectraidentification/spectraldatabase/SpectralMatchTask.java

```diff
@@ -210,10 +210,16 @@ private void addIdentities(Map<SpectralDBEntry, Double> matches) {
         }
 
         String compoundName = match.getKey().getField(DBEntryField.NAME).toString();
+        String shortName = compoundName;
```

Comment:
The check looks good

Reply:
what do you mean?

Reply:
I say, I like what you did. The position of my comment was a bit to early

Reply:
Ok. Yeah the GNPS mgf is weird. The adduct is positioned at the end of the name and there is no special field for this